sympy/integrals/tests/test_deltafunctions.py

```diff
@@ -35,8 +35,9 @@ def test_deltaintegrate():
     assert deltaintegrate(DiracDelta(x)**2, x) == DiracDelta(0)*Heaviside(x)
     assert deltaintegrate(y*DiracDelta(x)**2, x) == \
         y*DiracDelta(0)*Heaviside(x)
-    assert deltaintegrate(DiracDelta(x, 1)**2, x) is None
-    assert deltaintegrate(y*DiracDelta(x, 1)**2, x) is None
+    assert deltaintegrate(DiracDelta(x, 1)**2, x) == -DiracDelta(0, 2)*Heaviside(x)
```

Comment:
Maybe I misunderstand how deltaintegrate works. Is it used for both definite and indefinite integrals?